### PR TITLE
Added files for jdk11

### PR DESCRIPTION
**Pull request by aureliogrb**
Added the option of building JDK 11 as well as ServerJRE 8

### CODE REVIEW

File: OracleJava/java-11/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2015 Oracle and/or its affiliates. All rights reserved.
```
Comment (mriccell) [inline OracleJava/java-11/Dockerfile]: You need to update the date in your Copyright statement

File: OracleJava/java-11/build.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (mriccell) [inline OracleJava/java-11/build.sh]: You need to add Copy right and UPL
#Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
#
#Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.

File: OracleJava/java-11/Dockerfile (hunk 1 above)
Comment (aureliogrb) [reply OracleJava/java-11/Dockerfile]: Done.

File: OracleJava/java-11/build.sh (hunk 2 above)
Comment (aureliogrb) [reply OracleJava/java-11/build.sh]: Done

**Pull request merged**